### PR TITLE
Fix/php7.1 compatibility

### PR DESCRIPTION
**Pull request by boajer**
The reason is to improve compatibility with PHP 7.1

### CODE REVIEW

File: helpers/class.Environment.php [hunk 1]
```diff
@@ -84,15 +84,14 @@ public static function getOperatingSystem()
...
+     * @param  string $phpSyntax The PHP syntax to describe the variable.
      * @return int The size in bytes.
      */
     private static function toBytes($phpSyntax)
     {
-        $returnValue = (int) 0;
-
         $val = trim($phpSyntax);
         	$last = strtolower($val[strlen($val)-1]);
+        $val = (int)trim($val, $last);
```
Comment (bugalot) [inline helpers/class.Environment.php]: what if $first === $last ? :-)

Comment (boajer) [inline helpers/class.Environment.php]: Do you mean if value in php_ini is incorrect and looks like upload_max_filesize = 'M' ?

Comment (jbout) [inline helpers/class.Environment.php]: why not just use (int)substr($val, 0, -1) ?

**Comment by bugalot** [comment]
Hi @boajer,

In order to understand a little bit better the cases where $last = 'g' or 'm' or 'k', could you please just show me one or two invocation samples of the toBytes() method ? I'll had unit tests then and try with multiple versions of PHP.

Thanks,
Jérôme

**Comment by bugalot** [comment]
Finally got it... :)

**Pull request merged**